test/debug/config_test.rb

```diff
@@ -67,4 +67,142 @@ def test_config_show_frames_set
       end
     end
   end
+
+  class ShowSrcLinesTest < TestCase
+    def program
+      <<~RUBY
+      1| p 1
+      2| p 2
+      3| p 3
+      4| p 4
+      5| p 5
+      6| p 6
+      7| p 7
+      8| p 8
+      9| p 9
+      10| binding.b
+      11| p 11
+      12| p 12
+      13| p 13
+      14| p 14
+      15| p 15
+      RUBY
+    end
+
+    def test_show_src_lines_control_the_lines_displayed_on_breakpoint
+      debug_code(program) do
+        type 'config set show_src_lines 2'
+        type 'continue'
+        assert_line_text([
+          /9| p 9/,
+          /=>   10| binding.b/
+        ])
+
+        assert_no_line_text(/p 11/)
+        type 'continue'
+      end
+    end
+  end
+
+  class ShowFramesTest < TestCase
+    def program
+      <<~RUBY
+     1| def m1
+     2|   m2
+     3| end
+     4|
+     5| def m2
+     6|   m3
+     7| end
+     8|
+     9| def m3
+    10|   foo
+    11| end
+    12|
+    13| def foo
+    14|   binding.b
+    15| end
+    16|
+    17| m1
+      RUBY
+    end
+
+    def test_show_frames_control_the_frames_displayed_on_breakpoint
+      debug_code(program) do
+        type 'config set show_frames 2'
+        type 'continue'
+        assert_line_text([
+          /Object#foo at/,
+          /Object#m3 at/
+        ])
+
+        assert_no_line_text(/Object#m2/)
+        type 'continue'
+      end
+    end
+  end
+
+  class SkipPathTest < TestCase
+    def lib_file
+      <<~RUBY
+        def lib_m1
+          yield
+        end
+        def lib_m2
+          2
+        end
+      RUBY
+    end
+
+    def program(lib_file)
+      <<~RUBY
+     1| DEBUGGER__::CONFIG[:skip_path] = [/library/]
```

Comment:
I imagine this config will more likely to be set with code instead of command.

Reply:
Do not modify `CONFIG` hash because they will be frozen for Ractor support. Use `set_config skip_path: [...]`.

Reply:
Sorry I will prepare `CONFIG#[]=(key, val)` to change the configuration, so no need to change.

Reply:
Thanks for the update. I think this PR is ready for merge if no other changes are needed?